src/DotNetWorker/Context/Features/GrpcFunctionBindingsFeature.cs

```diff
@@ -16,6 +16,7 @@ internal class GrpcFunctionBindingsFeature : IFunctionBindingsFeature
 
         private IReadOnlyDictionary<string, object?>? _triggerMetadata;
         private IReadOnlyDictionary<string, object?>? _inputData;
+        private IDictionary<string, object?>? _outputData;
```

Comment:
Some of the logic here should probably be pulled to the base type (we may need to introduce one, though, doesn't have to be now) to lower the implementation cost.